packages/flutter/lib/src/gestures/converter.dart

```diff
@@ -104,90 +93,44 @@ class PointerEventConverter {
           );
           break;
         case ui.PointerChange.hover:
-          final bool alreadyAdded = _pointers.containsKey(datum.device);
+        case ui.PointerChange.move:
           final _PointerState state = _ensureStateForPointer(datum, position);
```

Comment:
Do we still need _ensureStateForPointer here? Isn't it now guaranteed by the protocol, that the PointerChange.add even will have created the state?